astropy/visualization/histogram.py

```diff
@@ -0,0 +1,60 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import absolute_import, division, unicode_literals
+
+import inspect
+import numpy as np
+from ..stats import histogram
+
+__all__ = ['hist']
+
+
+def hist(x, bins=10, ax=None, **kwargs):
+    """Enhanced histogram function
+
+    This is a histogram function that enables the use of more sophisticated
+    algorithms for determining bins.  Aside from the ``bins`` argument allowing
+    a string specified how bins are computed, the parameters are the same
+    as pylab.hist().
+
+    This function was ported from astroML: http://astroML.org/
+
+    Parameters
+    ----------
+    x : array_like
+        array of data to be histogrammed
+
+    bins : int or list or str (optional)
+        If bins is a string, then it must be one of:
```

Comment:
add an extra line break between each of these so it displays nicely when built by sphinx